flink-java/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -310,6 +310,9 @@ public int getFieldIndex(String fieldName) {
 
 	@Override
 	public TypeSerializer<T> createSerializer(ExecutionConfig config) {
+		if(config.isForceKryoEnabled()) {
+			return new GenericTypeInfo<T>(this.typeClass).createSerializer(config);
```

Comment:
I think it would be simpler to just do `return new KryoSerializer<T>(this.typeClass, config)`